core/templates/dev/head/pages/moderator-page/moderator-page.controller.ts

```diff
@@ -153,8 +153,8 @@ angular.module('oppia').directive('moderatorPage', [
             $http.post('/moderatorhandler/featured', {
               featured_activity_reference_dicts: activityReferencesToSave
             }).then(function() {
-              ctrl.lastSavedFeaturedActivityReferences = (
```

Comment:
As an FYI, it's totally fine to keep the parens.

Reply:
Okay